lib/files.cf

```diff
@@ -519,8 +519,8 @@ bundle edit_line insert_ini_section(name, config)
 {
   vars:
       # TODO: refactor once 3.7.x is EOL
```

Comment:
you fix the spelling, and you skip the TODO?

Reply:
So the naming was the refactoring needed? xD
Sry, missed that one.
@nickanderson If that is what's needed I can fix it.